### DISCUSSION TITLE
Legal risk

### DISCUSSION THREAD

**Discussion by JimWangKING** [atteneder/glTFast#509] [Q&A] [answered]
So for this plugin, Can I use it in compony business?

**Comment by atteneder** [comment, marked as answer]
Yes!
The package is licensed under the Apache 2.0 license. There's a few minor limitations, but it can be used for commercial purpose. Find all the details on the [license page](https://github.com/atteneder/glTFast/blob/main/LICENSE.md) and in case of doubt, check it with your lawyer.
